sunscreen_compiler_common/src/context.rs

```diff
@@ -229,9 +236,24 @@ where
      * Create a new [`Context`].
      */
     pub fn new(data: D) -> Self {
-        Self {
-            graph: CompilationResult::<O>::new(),
-            data,
+        #[cfg(not(feature = "debugger"))]
+        {
+            Self {
+                graph: CompilationResult::<O>::new(),
+                data,
+            }
+        }
+        
+        #[cfg(feature = "debugger")]
+        {
+            static GROUP_ID: AtomicU64 = AtomicU64::new(0);
```

Comment:
You shouldn't need atomics; a simple counter should suffice. `FheProgramContext` isn't thread-safe anyways.